Parsers/ASimWebSession/Test/Cisco_Meraki_ASimWebSession_SchemaTest.csv

```diff
@@ -0,0 +1,135 @@
+﻿Result
+"(1) Warning: Missing recommended field [ASimMatchingIpAddr]"
+"(1) Warning: Missing recommended field [DstHostname]"
+"(1) Warning: Missing recommended field [DvcIpAddr]"
+"(1) Warning: Missing recommended field [EventResultDetails]"
+"(1) Warning: Missing recommended field [EventUid]"
+"(1) Warning: Missing recommended field [SrcDomain]"
+"(1) Warning: Missing recommended field [SrcHostname]"
```

Comment:
are we not getting these recommended values from Meraki?

Reply:
EventUid we have mapped with _ResourceId but we are not getting value for other fields.